model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/hibernate/jsonb/JsonbType.java

```diff
@@ -148,9 +154,14 @@ private Object extractJson(CallableStatement statement, String name) throws SQLE
         }
     }
 
-    private static class JsonbJavaTypeDescriptor extends AbstractTypeDescriptor<Object> {
+    private static class JsonbJavaTypeDescriptor extends AbstractTypeDescriptor<Object> implements DynamicParameterizedType {
 
-        private static final JsonbJavaTypeDescriptor INSTANCE = new JsonbJavaTypeDescriptor();
+        private Class valueType;
+
+        @Override
+        public void setParameterValues(Properties parameters) {
+            valueType = ((ParameterType) parameters.get(PARAMETER_TYPE)).getReturnedClass();
```

Comment:
Where is PARAMETER_TYPE defined?

Reply:
There is the declaration: https://github.com/hibernate/hibernate-orm/blob/main/hibernate-core/src/main/java/org/hibernate/usertype/DynamicParameterizedType.java#L22